### ISSUE TITLE
no link works on ebay

### ISSUE THREAD

**Issue by GianniGi** [gorhill/uBlock#2968]
Filter issues **MUST NOT** be reported here. Read first: <https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md>

### Describe the issue

When clicking any link on ebay.* sites, nothing works, the url reloads only after you it F5
The problem persists if you disable ublock on the site. The problem goes away if you disable ublock on the site, and you click the options to "toggle the blocking of all popups" and "toggle the blocking of large media elements" and "toggle cosmetic filtering" and "toggle the blocking of remote fonts".
The problem goes away if you completely disable ublock.

### One or more specific URLs where the issue occurs

ebay.com
ebay.ie
any ebay

[URL(s) for issue on a specific site are **mandatory**]

### Screenshot in which the issue can be seen

[Screenshot(s) for visual issues are **mandatory**]

### Steps for anyone to reproduce the issue

1. Be as detailed as possible:
- go to ebay.com or any ebay
- click any link of a product, and nothing happens
- hit F5
- the pages reload and goes to the intended link

### Your settings

[If you fail to provide this info, I will mark the issue as invalid. Lists all settings which differs from default settings]

- OS/version: 
windows 8.1

- Browser/version: 
firefox 55.0.3

- uBlock Origin version: 
1.14.4

##### Your filter lists

Default filter lists + easy list + easy privacy + malware domain list + Peter Lowe’s Ad and tracking server + ita

**Comment by gomoku** [comment]
Duplicate of https://github.com/gorhill/uBlock/issues/2947
Have you ever used the search function in your life?: 
https://github.com/gorhill/uBlock/issues?utf8=%E2%9C%93&q=ebay

**Comment by GianniGi** [comment]
Yes I did: I searched and installed another adblocker.
